src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table-actions/table-actions.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, Input, OnInit } from '@angular/core';
```

Comment:
This should replace the current implementation of table-actions in all tables.
To do that, I would suggest creating a separate PR, since we don't want to include changes externals to Pools in this PR.
Only doing this will garante that it works in all situations and we don't need to change the code in the future.

Reply:
That was the plan ;)

Reply:
When I mentioned a separate PR, I was thinking of it being created/merged before this PR.
Are you currently working on it? or planning to?

Reply:
I can extract it to be merged before, to reduce the burden to review over 3k lines of code ;)